docs/cri/development.md

```diff
@@ -0,0 +1,53 @@
+# Getting Started for Developers
+
+This document provides steps to build `cri` plugin and test it locally.
+
+## Install Dependencies
+
+1. Follow this [guide](https://github.com/containerd/containerd/blob/master/BUILDING.md) and install required dependencies.
+2. Install [CNI](https://github.com/containernetworking/cni) by following its homepage.
+
+## Build and Install cri
+
+When building the containerd, cri is automatically being built unless you specify no_cri build tag. For more detail, see this [guide](https://github.com/containerd/containerd/blob/master/BUILDING.md#build-containerd)
+
+### Build Tags
+
+`cri` supports optional build tags for compiling support of various features.
+To add build tags to the make option the `BUILD_TAGS` variable must be set.
+
+```bash
+make BUILDTAGS='seccomp apparmor selinux'
+```
+
+| Build Tag | Feature                            | Dependency                      |
+|-----------|------------------------------------|---------------------------------|
+| seccomp   | syscall filtering                  | libseccomp development library  |
+| selinux   | selinux process and mount labeling | <none>                          |
+| apparmor  | apparmor profile support           | <none>                          |
+
+### Running a Kubernetes local cluster
+
+If you already have a working development environment for supported Kubernetes
+version, you can try `cri` in a local cluster:
+
+0. If containerd is already running as a daemon service, you need to stop it first.
+```bash
+sudo service containerd stop
+```
+
+1. Start the version of `containerd` with `cri` plugin that you built and installed
+above as root in a first terminal:
+```bash
+sudo containerd
```

Comment:
suggest running containerd in debug mode:
`sudo containerd -l debug`

Reply:
Sounds good.